scripts/tidb-test-cluster.sh

```diff
@@ -12,11 +13,12 @@ fi
 SCRIPT_PATH=$(realpath $(dirname "$0"))
 TEMPDIR=$(mktemp -d)
 DOCKER=$(which docker)
-TAG_VERSION="v6.1.0"
 SCRIPT_INIT=false
 MYIP="127.0.0.1"
```

Comment:
Please remove also `MYIP` as it's not longer used (if I'm reading it correctly).